package/whiteboard_hypercard/package

```diff
@@ -0,0 +1,31 @@
+# vim: set ft=sh:
+pkgname=whiteboard_hypercard
```

Comment:
I’d prefer to use dashes instead of underscores for separating parts of the package name. If you agree, you’ll also need to change the name of the directory in `package/` and the name of the Draft file.

```suggestion
pkgname=whiteboard-hypercard
```